source/templates/committer-pmc-results.txt

```diff
@@ -0,0 +1,14 @@
+To: private@[PROJECT].a.o
+Subject: [RESULT] [VOTE] New committer and PMC member candidate: Candiate Name
+
+The vote has now closed. The results are:
+
+Binding Votes:
+
++1 [TOTAL BINDING +1 VOTES]
+0 [TOTAL BINDING +0/-0 VOTES]
+-1 [TOTAL BINDING -1 VOTES]
+
```

Comment:
Those templates were always a bit annoying, because they were depersonalized, also those votes are always binding because they are happening in private@. So there is no need to mention that the votes are bining, also we are using slightly different template that is more personal, and might be a good idea to use it here

```suggestion
Binding Votes:

+1: N votes

* PMC Member 1
* PMC Member 2

0: N Votes:

* PMC Member 1
* PMC Member 2

-1 (Vetos): N votes

* PMC Member 1
* PMC Member 2
```

If that's fine. we should likely apply it elsewhere.

Reply:
Works for me. Does it mean that we should also have a "non-binding" section here @sebbASF ?

Reply:
Works for me too.

Reply:
Though I also prefer to mention names of people in the list - as it makes it personal - we are all humans (not only the people who we vote on but also the people who vote are not numbers)

Reply:
While I, also, like to see a list of names, very few PMCs actually do that. I guess the point of the template is to encourage behavior. Thanks.

Reply:
Understood. No problem - we can always apply our own "human-focused" standards in our PMC :)